master/getting-started/bare-metal/bare-metal.md

```diff
@@ -427,3 +430,280 @@ an ingress rule allowing access *to* port 999 and an egress rule allowing
 outbound traffic *from* port 999.  (Whereas for a connection tracked policy, it
 is usually enough to specify the ingress rule only, and then connection
 tracking will automatically allow the return path.)
+
+Because of how untracked policy is implemented, untracked ingress rules apply
+to all incoming traffic through a host endpoint - regardless of where that
+traffic is going - but untracked egress rules only apply to traffic that is
+sent from the host itself (not from a local workload) out of that host
+endpoint.
+
+## Pre-DNAT policy
+
+Policy for host endpoints can be marked as 'preDNAT'.  This means that rules in
+that policy should be applied before any DNAT (Destination Network Address
+Translation), which is useful if it is more convenient to specify Calico policy
+in terms of a packet's original destination IP address and port, than in terms
+of that packet's destination IP address and port after it has been DNAT'd.
+
+An example is securing access to Kubernetes NodePorts from outside the cluster.
+Traffic from outside is addressed to any node's IP address, on a known
+NodePort, and Kubernetes (kube-proxy) then DNATs that to the IP address of one
+of the pods that provides the corresponding Service, and the relevant port
+number on that port (which is usually different from the NodePort).
+
+As NodePorts are the externally advertised way of connecting to Services (and a
+NodePort uniquely identifies a Service, whereas an internal port number may
+not), it makes sense to express Calico policy to expose or secure particular
+Services in terms of the corresponding NodePorts.  But that is only possible if
+the Calico policy is applied before DNAT changes the NodePort to something
+else - and hence this kind of policy needs 'preDNAT' set to true.
+
+In addition to being applied before any DNAT, the enforcement of pre-DNAT
+policy differs from that of normal host endpoint policy in three key details,
+reflecting that it is designed for the policing of incoming traffic from
+outside the cluster:
+
+1. Only the ingress rules of a pre-DNAT policy are enforced.  If the policy has
+   any egress rules defined, they will have no effect.  (Standard connection
+   tracking is sufficient to allow return path traffic.)
+
+2. Pre-DNAT policy is enforced for all traffic arriving through a host
+   endpoint, regardless of where that traffic is going, and - in particular -
+   even if that traffic is routed to a local workload on the same host.
+   (Whereas normal host endpoint policy is skipped, for traffic going to a
+   local workload.)
+
+3. There is no 'default drop' semantic for pre-DNAT policy (as there is for
```

Comment:
I think this only applies if the packet is heading to a pod/service.  How about breaking this into cases for "packet terminated by host" (where the remaining normal policy then applies including default drop) and "packet forwarded on to workload or remote workload" (where the packet goes on its way)